sykefi/PRTR
# Add effect for resetting scroll on route change

## Description
Add a new effect for resetting scroll on route (pathname) change as suggested in the [React Router docs](https://reactrouter.com/web/guides/scroll-restoration). 

## This PR contains

- [ ] New feature(s)
- [ ] New/updated documentation
- [ ] Incomplete functionality

## PR checklist

- [x] I have reviewed the changes myself

## How to test?

1. Navigate to facility search results with smaller screen
2. Open a facility page from the list
3. The view should automatically scroll to the top of the facility page
